largest continuous subset/kadane.cpp

```diff
@@ -0,0 +1,56 @@
+// PROBLEM :: find the sum of the largest sum continuous array using KADANE'S ALGORITHM
+
+
+#include <iostream>
+#include <vector>
+using namespace std;
+
+
+void maxSumSubArray(vector<int>values)
+{
+	int max_so_far=values[0];
+	int curr_max=values[0];
+
+	int s=0,e=0;
+
+	int n=values.size();
+
+	for(int i=1;i<n;i++)
+	{
+		curr_max=max(values[i],curr_max+values[i]);
+
+		if(curr_max==values[i])
+		{
+			s=i;
+		}
+
+		if(curr_max>max_so_far)
+		{
+			max_so_far=curr_max;
+			e=i;
+		}
+	}
+
+	cout<<"the max contiguous sum is :: " << max_so_far<<"\n";
+	cout<<"the sequence starts from "<<s<<" and ends at "<<e<<"\n";
+	cout<<"and the values are :: ";
+	for(int i=s;i<=e;i++)
+		cout<<values[i]<<" ";
+}
+int main()
+{
+	int n;
```

Comment:
Change `n` to `size`, and change its type to [`size_t`](http://en.cppreference.com/w/cpp/types/size_t).